packages/react-docsite-components/src/components/CodepenComponent/CodepenComponent.tsx

```diff
@@ -53,8 +53,8 @@ const CodepenComponentBase: React.FunctionComponent<ICodepenProps> = props => {
       .filter(line => !!line)
       .join('\n');
 
-    const headContent = `${script('react@16.8.6/umd/react.development.js')}\n${script(
-      'react-dom@16.8.6/umd/react-dom.development.js',
+    const headContent = `${script('react@16/umd/react.development.js')}\n${script(
```

Comment:
Open to suggestions for what this should do--it could also just default to latest react, or I could try doing `require('react/package.json').version` or something